examples/server_profiles.py

```diff
@@ -99,6 +99,37 @@
 schema = oneview_client.server_profiles.get_schema()
 pprint(schema)
 
+# Server profile compliance preview
+print("\nGets the preview of manual and automatic updates required to make the server profile consistent "
+      "with its template.")
+schema = oneview_client.server_profiles.get_compliance_preview(server_profile_id)
+pprint(schema)
+
+# Get profile ports
+print("\nRetrieve the port model associated with a server hardware type and enclosure group")
+profile_ports = oneview_client.server_profiles.get_profile_ports(enclosureGroupUri=enclosure_group_uri,
+                                                                 serverHardwareTypeUri=server_hardware_type_uri)
+pprint(profile_ports)
+
+# Get profile ports
+# To run this example you must define a server hardware uri
+server_hardware_uri = None
+if server_hardware_uri:
+    print("\nRetrieve the port model associated with a server hardware")
+    profile_ports = oneview_client.server_profiles.get_profile_ports(serverHardwareUri=server_hardware_uri)
+    pprint(profile_ports)
+
+# Retrieve the error or status messages associated with the specified profile
+print("\nList profile status messages associated with a profile")
+messages = oneview_client.server_profiles.get_messages(server_profile_id)
+pprint(messages)
+
+# Transform an server profile
+print("\nTransform an existing profile by supplying a new server hardware type and/or enclosure group.")
+server_transformation = oneview_client.server_profiles.get_transformation(
+    server_profile_id, enclosureGroupUri=enclosure_group_uri, serverHardwareTypeUri=server_hardware_type_uri)
+pprint(server_transformation)
+
```

Comment:
In the transform example it would be cool if you could use the server_transformation response to edit the profile, something like oneview_client.server_profiles.update(resource=server_transformation, id_or_uri=server_transformation["uri"]).  This might make the transform example a little more complete.